internal/bminventory/inventory.go

```diff
@@ -148,7 +149,16 @@ const ignitionConfigFormat = `{
           "name": "root"
       },
       "contents": { "source": "data:,{{.AGENT_MOTD}}" }
-	}{{if .RH_ROOT_CA}},
+	},
+	{
+		"overwrite": true,
+		"path": "/root/.docker/config.json",
+		"mode": 644,
```

Comment:
I think this needs to be provided as decimal, ref: https://github.com/coreos/ignition/blob/master/doc/configuration-v3_1.md

Reply:
Done the same as MOTD from above.

Reply:
Fixed